roles/openshift_openstack/tasks/check-prerequisites.yml

```diff
@@ -105,3 +105,12 @@
   - "{{ openshift_openstack_node_flavor }}"
   - "{{ openshift_openstack_lb_flavor }}"
   - "{{ openshift_openstack_etcd_flavor }}"
+
+- name: Check Load Balancer options
+  fail:
+    msg: >
+      Only one of `openshift_openstack_use_lbaas_load_balancer` and
+      `openshift_openstack_use_vm_load_balancer` can be set at a time.
```

Comment:
nit: technically they can both be set; they just can't both be true, correct?

Reply:
Good point, fixed.